rdrf/registry/patients/models.py

```diff
@@ -196,6 +196,9 @@ def get_by_user_and_registry(self, user, registry_model):
             return qs.filter(user=user)
         if user.is_carer:
             return qs.filter(carer=user)
+        if user.is_parent:
+            parent_guardian = ParentGuardian.objects.filter(user=user).first()
```

Comment:
Shouldn't we check for None here ? Are we sure that ParentGuardian always exists in this case ?